ray-operator/hack/local_deploy.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+PROJECT_ROOT="${SCRIPT_DIR}/../.." # hack/ -> ray-operator/ -> kuberay/
+
+# Path to your Helm chart, relative to the PROJECT_ROOT (kuberay/)
+HELM_CHART_RELATIVE_PATH="helm-chart/kuberay-operator"
+HELM_CHART_PATH="${PROJECT_ROOT}/${HELM_CHART_RELATIVE_PATH}"
+
+RAY_OPERATOR_PATH="${PROJECT_ROOT}/ray-operator"
+
+# --- Configuration Variables ---
```

Comment:
These variables are not configurable. See previous comment https://github.com/ray-project/kuberay/pull/3863#discussion_r2211270168